### DISCUSSION TITLE
how the license.lic in mirth-client.jar generate,i compile the code with ant ,but there's none,i see others mirthconnect got one

### DISCUSSION THREAD

**Discussion by doodoocoder** [nextgenhealthcare/connect#6031] [Q&A]
i see other's mirthconnect has the license.lic in mirth-client.jar,how to gen ?i compile the code with ant,the mirth-client.jar generated has no  license.lic,can anybody tell me how to generate it?is there some ways to config withe ant or i miss some config.
![image](https://github.com/nextgenhealthcare/connect/assets/26543224/fb375767-539c-4628-ae1e-d2dcbd75d048)

**Comment by pacmano1** [comment]
Please do not post in the section unless you have a feature request or a bug. I am moving your question to discussions.
